x-pack/test/api_integration/apis/ml/trained_models/trained_models_list.ts

```diff
@@ -71,6 +71,16 @@ export default ({ getService }: FtrProviderContext) => {
           dfaRegressionN1.indices.length
         })`
       );
+
+      const downloadedModels = body.filter((v: any) => v.state !== 'notDownloaded');
+
+      downloadedModels.forEach((model: any) => {
+        if (ml.api.isInternalModelId(model.model_id)) {
+          expect(model.spaces).to.eql(['*']);
+        } else {
+          expect(model.spaces).to.eql(['default']);
+        }
```

Comment:
Rather than having these expects inside an if statement, it could be written like this:
```suggestion
const expectedSpaces = ml.api.isInternalModelId(model.model_id) ? ['*'] : ['default'] 
expect(model.spaces).to.eql(expectedSpaces)
```